about/about-serverless.adoc

```diff
@@ -25,11 +24,49 @@ For additional information about the {ServerlessProductName} life cycle and supp
 
 // Add something about CLI tools
 
+== Components of {ServerlessProductName} 
+
+=== Knative Serving
+Knative Serving builds on Kubernetes to support deploying and serving of applications and functions as serverless containers. Serving simplifies the application deployment, dynamically scales based on in incoming traffic and supports custom rollout strategies with traffic splitting. Other features include:
+
+* Simplified deployment of serverless containers
+* Traffic-based auto-scaling, including scale-to-zero
+* Routing and network programming
+* Point-in-time application snapshots and their configurations
+
+=== Knative Eventing
+Knative Eventing provides a platform that offers composable primitives to enable late-binding event sources and event consumers. Knative Eventing supports the following architectural cloud-native concepts:
+
+* Services are loosely coupled during development and deployed independently to production
+* A producer can generate events before a consumer is listening, and a consumer can express an interest in an event or class of events that are not yet being produced.
+* Services can be connected to create new applications without modifying producer or consumer, and with the ability to select a specific subset of events from a particular producer.
+
+=== {FunctionsProductName}
+{FunctionsProductName} allows you to write functions that are deployed as Knative Services and take advantage of Knative Serving and Eventing. {FunctionsProductName} bring greater efficiency, more scalability and faster development to facilitate rapid go-to-market. Other features include:
+
+* Build strategies including Source-to-Image (S2I) and Buildpacks
+* Multiple runtimes, including Node.js, Quarkus and Go
+* Local developer experience through the kn CLI
+* Project templates
+* Support for receiving CloudEvents and plain HTTP requests
+
+=== {ServerlessLogicProductName}
+{ServerlessLogicProductName} allows you to define declarative workflow models (YAML, JSON) that orchestrate event-driven, serverless applications and lets you visualize the execution of workflows, which simplifies debugging and optimization. It also includes built-in error handling and fault tolerance, making it easier to handle errors and exceptions that occur during the execution of a workflow. 
+{ServerlessLogicProductName} implements the link:https://serverlessworkflow.io/[CNCF Serverless Workflow specification].
+
+=== Knative CLI
+The Knative client `kn` allows you to create Knative resources from the command line or from within Shell scripts. With its extensive help pages and autocompletion support, it frees you from memorizing the detailed structure of the Knative resource schemas. `kn` features include:
+
+* Full support for managing all features of Knative Serving: Services, Revisions and Routes
+* Support for managing Knative Eventing entities: Sources, Brokers, Triggers, Channels and Subscriptions
```

Comment:
This would scan better and have a more parallel structure if you used subbullets to create a nested list.

See [Nested unordered lists](https://docs.asciidoctor.org/asciidoc/latest/lists/unordered/#nested-unordered-list) (Asciidoc docs)